src/dmd/mars.d

```diff
@@ -486,6 +486,12 @@ private int tryMain(size_t argc, const(char)** argv)
         message("binary    %s", global.params.argv0);
         message("version   %s", global._version);
         message("config    %s", global.inifilename ? global.inifilename : "(none)");
+        // Print DFLAGS environment variable
+        OutBuffer buf;
+        foreach (flag; dflags.asDArray)
+            buf.printf("%s ", flag);
```

Comment:
I think individual dflags may contain spaces.  If so, we may want to surround them with quotes when we see that they contain whitespace.

Reply:
So you want me to detect whether there's a space in the flag and then use quotes for it?
Most flags don't use spaces and it would look pretty ugly if this is done by default.

Reply:
DFLAGS=-I"path to foo" -Ibar
=>
`'-Ipath to foo' -Ibar`
so only `''` where needed;

also, using
`-Ipath\ to\ foo -Ibar`
is another option

Reply:
Actually, a better solution might just be to print the original unprocessed value of `DFLAGS` because that one will already be quoted properly